OxfordBodleian/Juel-Jensen/BDLaethg37.xml

```diff
@@ -519,6 +520,93 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             </handNote>
                         </handDesc>
 
+                        <decoDesc>
+                          <summary>The prayers in the manuscript are prefaced by a portrait of
+                          their compiler. These are without captions, but their position in relation to each prayer,
```

Comment:
I took a few seconds to understand "compiler" in this context - at first, I thought the scribe of the manuscript was meant. I got it then :) but wonder if there is a better term? In German, "Betende" would work but "praying person" sounds a bit weird...
In any case, maybe it is obvious to everyone else, then let'ss keep it as it is!